x-pack/plugins/fleet/public/applications/integrations/sections/epm/components/package_list_grid.tsx

```diff
@@ -123,12 +130,31 @@ export const PackageListGrid: FunctionComponent<Props> = ({
             <EuiSearchBar
               query={searchTerm || undefined}
               box={{
+                'data-test-subj': 'epmList.searchBar',
                 placeholder: i18n.translate('xpack.fleet.epmList.searchPackagesPlaceholder', {
                   defaultMessage: 'Search for integrations',
                 }),
                 incremental: true,
               }}
               onChange={onQueryChange}
+              toolsRight={
+                selectedCategory ? (
+                  <div>
+                    <EuiBadge
+                      color="accent"
+                      iconType="cross"
+                      iconSide="right"
+                      iconOnClick={() => {
+                        setSelectedCategory('');
+                      }}
+                      iconOnClickAriaLabel="Remove category"
+                      data-test-sub="epmList.categoryBadge"
+                    >
+                      {selectedCategory}
```

Comment:
I am wondering if we could use the same display name to show on the badge as in Category list, e.g. Elastic Agent instead of elastic_agent

Reply:
It's probably showing the id, I'll see how to get the name